documentation/docs/libraries/java/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx

```diff
@@ -0,0 +1,14 @@
+import CodeBlock from '@theme/CodeBlock';
+import CodeSample from '!!raw-loader!../../../../../../../wallet/bindings/java/examples/src/CheckUnlockConditions.java';
+
+<CodeBlock className="language-java">
+    {CodeSample}
+</CodeBlock>
+
+### Run the Example
```

Comment:
You sometimes have this heading and sometimes not, bit weird

Reply:
That happens if you do copy paste :trollface:

Reply:
In the existing files, it's only in the Rust examples, not in the bindings

Reply:
@Dr-Electron when consistency :trollface: ?

Reply:
😅

Reply:
:wen: look at your docs from time to time? :trollface: